sample-compose/app/src/androidTest/java/co/nimblehq/sample/compose/ui/screens/home/HomeScreenTest.kt

```diff
@@ -0,0 +1,80 @@
+package co.nimblehq.sample.compose.ui.screens.home
+
+import androidx.activity.compose.setContent
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.ComposeContentTestRule
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.test.rule.GrantPermissionRule
+import co.nimblehq.sample.compose.domain.model.Model
+import co.nimblehq.sample.compose.domain.usecase.UseCase
+import co.nimblehq.sample.compose.test.TestDispatchersProvider
+import co.nimblehq.sample.compose.ui.AppDestination
+import co.nimblehq.sample.compose.ui.screens.MainActivity
+import co.nimblehq.sample.compose.ui.theme.ComposeTheme
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.flow.flowOf
+import org.junit.*
+import org.junit.Assert.assertEquals
+
+class HomeScreenTest {
+
+    @get:Rule
+    val composeRule = createAndroidComposeRule<MainActivity>()
+
+    /**
+     * More test samples with Runtime Permissions https://alexzh.com/ui-testing-of-android-runtime-permissions/
+     */
+    @get:Rule
+    val permissionRule: GrantPermissionRule = GrantPermissionRule.grant(
+        android.Manifest.permission.CAMERA
+    )
+
+    private val mockUseCase: UseCase = mockk()
+
+    private lateinit var viewModel: HomeViewModel
+    private var expectedAppDestination: AppDestination? = null
+
+    @Before
+    fun setup() {
```

Comment:
(Optional) For consistency with ViewModel tests
```suggestion
    fun setUp() {
```